### PR TITLE
New healthcheck scripts to test various OpenShift Streams use-cases

### PR DESCRIPTION
**Pull request by DuncanDoyle**

### CODE REVIEW

**Comment by DuncanDoyle** [comment]
@wtrocki Initial PR. No need to merge now, but wanted to share this early. I will be adding a couple of other scripts in the next couple of days, specifically to produce and consume messages via kcat.

File: scripts/create-service-account.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+function usage {
+      echo "\n"
+      echo "Usage: create-service-account.sh [args...]"
+      echo "where args include:"
+      echo "    -s              The name of the service account to be created."
+      echo "    -o              Offline authentication token. Browser based login will be used if not specified."
+}
+
+#Parse the params
+while getopts ":s:o:h" opt; do
```
Comment (wtrocki) [inline scripts/create-service-account.sh:32]: Wow. This looks like bash based CLI to me :)

Comment (DuncanDoyle) [reply scripts/create-service-account.sh:32]: It sort of is ;-) But it works really nicely 👍

**Comment by DuncanDoyle** [comment]
@wtrocki Initial impl "sort of" completed. There now also is a script to test message production and consumption. One caveat is that the scripts that create topics require a browser-based login, as those commands don't seem to support offline token based logins.

But I think this is good as an initial starting point.

**Comment by wtrocki** [comment]
> browser-based login, as those commands don't seem to support offline token based logins.

This is fine. For time being requirement will be to login beforehand. In couple weeks we will have pure token support.
Reviewing

**Pull request merged**